components/user-mgt/org.wso2.carbon.user.mgt/pom.xml

```diff
@@ -70,6 +70,14 @@
             <groupId>org.apache.poi.wso2</groupId>
             <artifactId>poi-ooxml</artifactId>
         </dependency>
+        <dependency>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+        </dependency>
```

Comment:
Why are we adding new dependency here? I think we already have log, related dependency. Can't we use them